myhpi/tenca_django/templates/mailman/unsubscription_message.html

```diff
@@ -0,0 +1,9 @@
+Dear user,
+
+Please confirm your *unsubscription* from {{fqdn_listname}}:
+{{action_link}}
+
+If you did not request this, please report abuse by visiting the following link:
+$action_abuse_link
```

Comment:
Why is this not in `{{ }}`?

Reply:
forgot to replace from the old syntax ^^